test/IntegrationTests/FileVectorizer/test_deletefiles.py

```diff
@@ -0,0 +1,363 @@
+        ]
+        TEST_DIR = os.path.join(os.path.dirname(__file__), "FilesTesting")
+
+        if not os.path.exists(TEST_DIR):
+            self.fail(f" Test directory '{TEST_DIR}' not found")
+
+        customers = []
+        for i in range(5):
+            customer_data = add_customer(f"test_org_{i}")
+            token = create_test_token(org_id=customer_data.get("org_id"), org_role="org:admin")
+            headers = {'Authorization': f'Bearer {token}'}
+            files_to_upload = TEST_FILES[i * 10:(i + 1) * 10]  # Properly split 10 files per customer
+
+            customers.append({
+                "customer_guid": customer_data["customer_guid"],
+                "org_id": customer_data["org_id"],
+                "token": token,
+                "headers": headers,
+                "files_to_upload": files_to_upload,
+                "uploaded_files": []
+            })
+
+        upload_url = f"{self.BASE_URL}/uploadFile"
+        delete_url = f"{self.BASE_URL}/deletefile"
+        embedding_status_url = f"{self.BASE_URL}/file/list"
+        deletion_status_url = f"{self.BASE_URL}/files/deletionstatus"
+
+        logger.info("====Uploading 50 Files (10 per customer) ====")
+        for idx, customer in enumerate(customers):
+            logger.info(f"Customer {idx + 1}: Uploading {len(customer['files_to_upload'])} files...")
+
+            for filename in customer["files_to_upload"]:
+                file_path = os.path.join(TEST_DIR, filename)
+                if not os.path.exists(file_path):
+                    logger.warning(f"File {filename} not found, skipping")
+                    continue
+
+                with open(file_path, 'rb') as file:
+                    response = requests.post(
+                        upload_url,
+                        files={"file": (filename, file)},
+                        headers=customer["headers"]
+                    )
+
+                self.assertEqual(response.status_code, 200, f"❌ Failed to upload {filename}")
+                file_id = response.json().get("file_id")
+                customer["uploaded_files"].append((filename, file_id))
+                logger.info(f"Uploaded {filename} (ID: {file_id})")
+
+            self.assertEqual(
+                len(customer["uploaded_files"]), 10,
+                f"Customer {idx + 1} has {len(customer['uploaded_files'])} files (expected 10)"
+            )
+
+        logger.info("==== Monitoring Embedding Status ====")
+        for idx, customer in enumerate(customers):
+            logger.info(f"Customer {idx + 1}: Checking embedding progress...")
+            start_time = time.time()
+
+            while True:
+                response = requests.get(
+                    embedding_status_url,
+                    headers=customer["headers"],
+                    params={"page": 1, "page_size": 100}
+                )
+                self.assertEqual(response.status_code, 200)
+
```

Comment:
Write an assert to check that the number of files in the response is equal to 10